src/libsync/connectionvalidator.cpp

```diff
@@ -252,7 +253,17 @@ void ConnectionValidator::slotUserFetched(const QVariantMap &json)
     QString user = json.value("ocs").toMap().value("data").toMap().value("id").toString();
     if (!user.isEmpty()) {
         _account->setDavUser(user);
+
+        AvatarJob *job = new AvatarJob(_account, this);
+        QObject::connect(job, SIGNAL(avatarPixmap(QPixmap)), this, SLOT(slotAvatarPixmap(QPixmap)));
+
```

Comment:
Missing  
```
    job->setTimeout(timeoutToUseMsec);
```

The default timeout is 5 minutes, and that's way to long for the connection validator which must finish much earlier in case of bad connection.


(And actually, since we increased the amount of jobs, timeoutToUseMsec might need to be lowered too.   But I'm wondering if it would be possible to fetch the capabilities, user name, and the avatar in parallel.)